src/optimizer/loadOptimizers.ts

```diff
@@ -60,6 +74,7 @@ export async function loadOptimizers(
   let ordered: string[];
   try {
     ordered = toposort.array(nodes, edges).reverse();
+    console.log(JSON.stringify(ordered));
```

Comment:
We want to take this out (good news is though that it does confirm for me that "simplify_array_indexing" is running last).

Reply:
Removed!